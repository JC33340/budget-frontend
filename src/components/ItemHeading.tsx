type ItemHeadingType = {
    text: string;
    size?: 'md' | 'sm';
};

const ItemHeading = ({ text, size = 'md' }: ItemHeadingType) => {
    return (
        <p
            className={`${size === 'md' ? 'text-3xl' : 'text-2xl'} text-blue font-medium text-center`}
        >
            {text}
        </p>
    );
};

export default ItemHeading;
