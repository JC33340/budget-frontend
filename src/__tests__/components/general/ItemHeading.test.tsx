import ItemHeading from '../../../components/ItemHeading';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';

test('snapshot test', () => {
    render(<ItemHeading text="test-heading" />);
    const itemHeading = screen.getByTestId('itemHeading-testid');
    expect(itemHeading).toMatchSnapshot();
});
