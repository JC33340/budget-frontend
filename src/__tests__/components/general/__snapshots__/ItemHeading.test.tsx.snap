// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<p
  class="text-3xl text-blue font-medium text-center"
  data-testid="itemHeading-testid"
>
  test-heading
</p>
`;
